packages/mds-transaction-api/handlers/get-transactions.ts

```diff
@@ -177,3 +184,82 @@ export const GetTransactionsHandler = async (
     next(error)
   }
 }
+export const GetTransactionsAsCsvHandler = async (
+  req: TransactionApiGetTransactionsRequestWithPickColumns,
+  res: TransactionApiGetTransactionsAsCsvResponse,
+  next: express.NextFunction
+) => {
+  try {
+    const { provider_id } = parseRequest(req).single({ parser: String }).query('provider_id')
+    const order = getOrderOption(req)
+    const {
+      start_timestamp,
+      end_timestamp,
+      limit = 10
+    } = parseRequest(req).single({ parser: Number }).query('start_timestamp', 'end_timestamp', 'limit')
+    const PICKABLE_COLUMNS = <const>[
+      'transaction_id',
+      'provider_id',
+      'device_id',
+      'timestamp',
+      'fee_type',
+      'amount',
+      'receipt.receipt_id',
+      'receipt.timestamp',
+      'receipt.origin_url',
+      'receipt.receipt_details'
+    ]
```

Comment:
What is this list based on, or is it just arbitrary? I think we may want to make this _any_ possible property of a Transaction, so a `DeepPickPaths<TransactionDomainModel>`

Reply:
It's just a listing of all the things in `TransactionDomainModel` and, recursively, `ReceiptDomainModel`. (Excluding whatever properties `receipt_details` may or may not have)

Reply:
Added a few more columns, including everything in the epic except for policy name